espnet2/layers/stft.py

```diff
@@ -3,13 +3,16 @@
 import librosa
 import numpy as np
 import torch
+from packaging.version import parse as V
 from torch_complex.tensor import ComplexTensor
 from typeguard import check_argument_types
 
 from espnet2.enh.layers.complex_utils import to_complex
 from espnet2.layers.inversible_interface import InversibleInterface
 from espnet.nets.pytorch_backend.nets_utils import make_pad_mask
 
+is_torch_1_10_plus = V(torch.__version__) >= V("1.10.0")
```

Comment:
You don't need to add it because current ESPnet already dropped support for PyTorch versions before 1.12.1.

Reply:
See line 93 of `espnet2/layers/stft.py`. Add this check to use native support for FFT and STFT on all CPU targets including ARM.